k9mail/src/main/java/security/planck/ui/passphrase/PassphrasePresenter.kt

```diff
@@ -13,6 +13,9 @@ import security.planck.ui.PassphraseProvider
 import timber.log.Timber
 import javax.inject.Inject
 
+private const val ACCEPTED_SYMBOLS = """@\$!%*+\-_#?&\[\]\{\}\(\)\.:;,<>~"'\\/"""
+private const val PASSPHRASE_REGEX = """^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[$ACCEPTED_SYMBOLS])[A-Za-z\d$ACCEPTED_SYMBOLS]{12,}$"""
```

Comment:
is this the BSI rule? Should we make this configurable?

Reply:
I don't think so, like the symbols accepted for the passhprase. Roberto passed me some symbols as example and I added a few more for completion. We can always add more.

Reply:
ok, if we get the requirement to make this configurable we can always do it in another stroy